CODEOWNERS

```diff
@@ -1,2 +1,34 @@
-*       @infinispan/community-committers
-
+*                      @infinispan/community-committers
+/api/                  @karesti @tristantarrant
+/archetypes/           @ryanemerson
+/cli/                  @tristantarrant
+/client/hotrod-client  @wburns
+/client/rest-client/   @tristantarrant
+/core/                 @jabolina @pruivo @wburns
+/counter/              @pruivo
+/documentation/        @ashwin-rh @andyuk1986 @Crumby
```

Comment:
I think @ashwin-rh need to have write permissions in order to be added here:

```

Unknown owner on line 9: make sure @ashwin-rh exists and has write access to the repository

/documentation/        @ashwin-rh @andyuk1986 @Crumby

```

Reply:
I've used @theashiot instead